xml/System.Windows.Threading/Dispatcher.xml

```diff
@@ -1063,8 +1063,8 @@
       </Parameters>
       <Docs>
         <param name="method">A delegate to a method that takes parameters specified in <paramref name="args" />, which is pushed onto the <see cref="T:System.Windows.Threading.Dispatcher" /> event queue.</param>
-        <param name="timeout">The maximum amount of time to wait for the operation to complete.</param>
-        <param name="args">An array of objects to pass as arguments to the given method. Can be <see langword="null" />.</param>
+        <param name="timeout">The minimum amount of time to wait for the operation to start. Once the operation has started, it will complete before this method returns.</param>
```

Comment:
- I don't think that this definition is accurate -- it suggests a minimum amount of time must elapse before the operation can start. The timeout typically defines the amount of time that can elapse before the operation is aborted if it has not started.   
- The sentence "Once the operation has started..." doesn't seem relevant to the timeout parameter.

Reply:
- Ron is right.  Both the old wording and the new are wrong (and so are the comments in the code).  The correct wording is "The maximum amount of time to wait for the operation to start". 
- The point of the second sentence (lifted verbatim from code comment) is to clarify that the timeout's job is to abandon operations that spend too long waiting in the queue, not to interrupt operations that take too long to run.  That's worth mentioning.
- Same remarks apply to all the methods being changed here.